website/pages/admin.html

```
@@ -1,99 +1,211 @@
 <!doctype html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="icon" href="../assets/images/logo.png" height="70px" width="100px">
-    <title>Admin Dashboard | 100 Days of Web Dev</title>
-    <link rel="stylesheet" href="../styles/index.css">
-    <link rel="stylesheet" href="../styles/pages/admin.css">
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.1/css/all.min.css" />
-    <script src="../scripts/components/sidebar-loader.js"></script>
+<meta charset="UTF-8">
```

HTMLHint (1.9.0)

[warning] The empty tag : [ meta ] must be self closed.

(tag-self-close)